lib/features/Auth/login/data/models/user.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/foundation.dart';
+
+/// This data class represents all data necessary to create a login
+class Login {
```

Comment:
if its a user file
name the class User
```suggestion
class User {
```
or name the file login_profile.dart
and
```suggestion
class LoginProfile {
```
keep specific file names